examples/vue.test.js

```diff
@@ -2600,6 +2600,63 @@ describe('Vue', function() {
       // acquit:ignore:end
     });
   });
+  it('vue-websocket', async function() {
+    // acquit:ignore:start
+    this.timeout(10000);
+    const browser = await puppeteer.launch({ headless: false });
+    const page = await browser.newPage();
+    const fn = function() {
+    // acquit:ignore:end
+      const app = new Vue({
+        data: function(){
+          return{
```

Comment:
Indent is a bit nasty here, just do `data => ({ time: null })`. There's no need to make `connection` a data property, because you aren't going to use `connection` in a template